docroot/sites/now.uiowa.edu/modules/now_core/now_core.module

```diff
@@ -41,6 +79,30 @@ function now_core_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   }
 }
 
+/**
+ * Alter the fields used to represent an entity in the IEF table.
+ *
+ * @param array $fields
+ *   The fields, keyed by field name.
+ * @param array $context
+ *   An array with the following keys:
+ *   - parent_entity_type: The type of the parent entity.
+ *   - parent_bundle: The bundle of the parent entity.
+ *   - field_name: The name of the reference field on which IEF is operating.
+ *   - entity_type: The type of the referenced entities.
+ *   - allowed_bundles: Bundles allowed on the reference field.
+ *
+ * @see \Drupal\inline_entity_form\InlineFormInterface::getTableFields()
```

Comment:
Is this necessary? Or could this be `{@inheritdoc}`?

Reply:
Shortened it to

```
/**
 * Implements hook_inline_entity_form_table_fields_alter().
 */
```

Does that work?